src/common/Network/PacketDef/Ipcs.h

```diff
@@ -107,7 +107,7 @@ namespace Sapphire::Network::Packets
     MailLetterNotification = 0x013A, // updated 5.0
 
     MarketBoardItemListingCount = 0x0125, // updated 4.5
-    MarketBoardItemListing = 0x0126, // updated 4.5
+    MarketBoardItemListing = 0x013C, // updated 5.0
```

Comment:
`MarketBoardItemListingCount` is probably 13b and `MarketBoardItemListingHistory` is would be 13d given they're in the same group

Reply:
Confirmed ItemListingCount, ListingHistory is not 13D. Filtering the packets is giving me a garbage opcode for ListingHistory, still need to fix that bug on my end

Reply:
huh, surprising that it's moved, wonder what's there now